GoogleForCreators/web-stories-wp
# Story Locking: Enable by default

## Context

Enable enablePostLockingTakeOver feature by default.



## Summary

Sets feature flag to default



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

Test instructions via https://github.com/GoogleForCreators/web-stories-wp/pull/12373 

1. Create a story.
2. Add some content.
3. Save as draft.
4. Open another browser.
5. Login, as a differnet user.
6. Open story ( from previous ) in editor.
7. Click Take over story.
8. In first browser, make changes to story. DO NOT SAVE.
9. Wait 1 minute 30 seconds.
10. Go back to second browser.
11. Refresh browser.
12. See message that there is a newer autosave.


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?
No



### Does this PR have a legal-related impact?
No



## Checklist




---



Fixes #12326
